spec/requests/api/v1/dqt_records_spec.rb

```diff
@@ -64,5 +64,15 @@
         expect(response.status).to eq 401
       end
     end
+
+    context "using token from different scope" do
+      let(:other_token) { EngageAndLearnApiToken.create_with_random_token! }
```

Comment:
I find it surprising that E&L is not considered internal

Reply:
I don't know the engage and learn api endpoints so not sure if we want it to have private access. if it gets changed it will break this test. I can't actually change to `ApiToken.create_with_random_token!` as I get the following error
```
ActiveRecord::RecordInvalid:
       Validation failed: Lead provider must exist
```
I don't really want to write code to workaround that and would prefer to use a token I know at this moment in time has `private_api_access` set to false